src/main/kotlin/io/github/inductiveautomation/kindling/idb/tagconfig/model/TagConfig.kt

```diff
@@ -0,0 +1,153 @@
+package io.github.inductiveautomation.kindling.idb.tagconfig.model
+
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.JsonArray
+import kotlinx.serialization.json.JsonElement
+import kotlinx.serialization.json.JsonNull
+import kotlinx.serialization.json.JsonObject
+import kotlinx.serialization.json.JsonPrimitive
+import kotlinx.serialization.json.JsonTransformingSerializer
+import kotlinx.serialization.json.jsonObject
+
+@Serializable
+data class TagConfig(
```

Comment:
Woof, this class is rough. Not a fault on your code, more on Ignition's :laughing: